cadence/contracts/ISPOManager.cdc

```diff
@@ -353,9 +356,15 @@ pub contract ISPOManager {
         }
 
         pub fun withdrawAdminFlowRewards(): @FungibleToken.Vault {
+            if self.stakingRewardsVault == nil {
+                self.stakingRewardsVault <-! FlowToken.createEmptyVault()
+            }
             let stakingRewardsVaultRef: &FungibleToken.Vault? = &self.stakingRewardsVault as &FungibleToken.Vault?
             for key in self.delegators.keys {
-                let delegatorRecordRef: &ISPOManager.DelegatorRecord = self.borrowDelegatorRecord(delegatorId: key)
+                let delegatorRecordRef: &ISPOManager.DelegatorRecord = self.borrowDelegatorRecord(delegatorId: key)!
```

Comment:
I think you forgot ! in the end, this will panic before reaching the `if` if delegator record is null

Reply:
fixed